t/script/inactive.t

```diff
@@ -15,6 +15,20 @@ $t = DateTime->now->subtract(months => 4);
 $user_inactive->last_active($t);
 $user_inactive->update;
 
+for (my $m = 1; $m <= 12; $m++) {
+    my $t = DateTime->new(year => 2017, month => $m, day => 1, hour => 12);
+    $mech->create_problems_for_body(1, 2237, 'Title', {
+        dt => $t,
+        lastupdate => "$t",
+        state => $m % 2 ? 'fixed - user' : 'confirmed',
+    });
+}
+
+$in->reports;
+
+my $count = FixMyStreet::DB->resultset("Problem")->search({ user_id => $user->id })->count;
+is $count, 6, 'Six non-anonymised';
```

Comment:
This should probably test anonymising updates and new updates on anonymised reports.